tests/foreman/api/test_errata.py

```diff
@@ -199,17 +199,21 @@ def test_positive_install_multiple_in_host(
             'activationkey-id': ak.id,
         }
     )
-    # Install katello-ca, register content-host, enable all repos:
-    rhel_contenthost.install_katello_ca(target_sat)
-    rhel_contenthost.register_contenthost(function_org.name, ak.name, function_lce.name)
+    rhel_contenthost.register(
+        org=function_org,
+        activation_keys=ak.name,
+        lifecycle_environment=function_lce,
+        target=target_sat,
+        force=True,
+        loc=None,
+    )
     assert rhel_contenthost.subscribed
     # Installing outdated custom packages:
     for package in constants.FAKE_9_YUM_OUTDATED_PACKAGES:
         rhel_contenthost.run(f'yum remove -y {str(package.split("-", 1)[0])}')
         assert rhel_contenthost.run(f'yum install -y {package}').status == 0
         assert rhel_contenthost.run(f'rpm -q {package}').status == 0
     rhel_contenthost.add_rex_key(satellite=target_sat)
```

Comment:
when using Global registration REX is setup as default, which includes copying key as well, so this step will no longer required
```suggestion
```